model/geographyHomepage/model.go

```diff
@@ -0,0 +1,20 @@
+package geographyHomepage
+
+import "github.com/ONSdigital/dp-frontend-models/model"
+
+// Page contains the data re-used on each page as well as the data for the current page
+type Page struct {
+	model.Page
+	Data GeographyList `json:"data"`
+}
+
+// GeographyList represents the data on the Geography list page
+type GeographyList struct {
+	Items []GeographyDataType `json:"items"`
+}
+
+// GeographyDataType represents the Type of data of the Geography list page
+type GeographyDataType struct {
```

Comment:
We probably don't need to repeat `Geography` and similar to above, can we be more specific than `DataType`, for example it's an `AreaType`?